packages/react-apollo/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```

Comment:
nit, maybe personal preference, I prefer the `"vite --host"` command as my default dev script.

Reply:
I wonder whether that flag should be the default across all projects, when do you find yourself needing this functionality?

Reply:
If the project I'm working on has a mobile view, I like to test it on an actual device a few times. Some projects also have mobile-specific functionality that only works on a real device. I'm not sure that it should be the default, but I've found that it's what my dev scripts end up being. There may be a downside that I don't know about, though.

Reply:
My understanding is that it's technically more secure to default the dev server to `localhost` to avoid any wider network access (elsewhere on your local network, or externally if port forwarding is configured on your router).

Admittedly, this is probably a fairly minor concern. But it's partially why I hesitate to include it here as the default.

Reply:
Adding another script is a good idea, thanks! 👍